groclient/client.py

```diff
@@ -902,7 +902,7 @@ def get_top(self, entity_type, num_results=5, **selection):
             self.access_token, self.api_host, entity_type, num_results, **selection
         )
 
-    def get_df(self, show_revisions=False, index_by_series=False, include_names=False):
+    def get_df(self, show_revisions=False, show_available_date=False, index_by_series=False, include_names=False):
```

Comment:
I know it was already out of date... but would you mind updating the doc string? :)

Reply:
good catch! updated